ORK1Kit/ORK1Kit/Common/ORK1WebViewStepViewController.m

```diff
@@ -112,12 +112,12 @@ - (instancetype)initWithStep:(ORK1Step *)step {
             [weakSelf userContentController:userContentController didReceiveScriptMessage:scriptMessage];
         };
         
-        _webView = [[ORK1WebViewPreloader shared] webViewForKey:step.identifier];
+        _webView = [[ORK1WebViewPreloader shared] webViewForKey:step.identifier baseURL:[self webViewStep].baseURL];
         [_webView.configuration.userContentController addScriptMessageHandler:_scriptMessageHandlerImpl name:@"ResearchKit"];
         _webView.frame = self.view.bounds;
         _webView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
         _webView.navigationDelegate = self;
-        [_webView loadHTMLString:[self webViewStep].html baseURL:nil];
+        [_webView loadHTMLString:[self webViewStep].html baseURL:[self webViewStep].baseURL];
```

Comment:
In ORK1WebViewStepViewController.stepDidChange and ORKWebViewStepViewController.stepDidChange it calls `[_webView loadHTMLString:[self webViewStep].html baseURL:nil];` without a baseURL. Does it need it there?

Reply:
Good catch, thank you!